track changes (#94)
DevExpress/dotnet-eud: interface-elements-for-desktop/articles/rich-text-editor/track-changes/enable-track-changes.md

# Enable Track Changes

> The Rich Text Editor allows you to use Track Changes to make revisions in a document.

Reviewer: to use Track Changes to make revisions? The 'Track Changes' feature allows you to keep track of the changes, to see which changes were made

Author: changed

## Turn On Track Changes

> On the **Review** [tab](text-editor-ui/ribbon-interface.md), in the **Tracking** group, click the **Track Changes** button.

Reviewer: Same here - Why do you write it in this order?

Author: I follow the Microsoft Manual of Style

> _Microsoft style_
> Under Table Tools, on the Layout tab, in the Data group, click Repeat Header Rows.
> _Not Microsoft style_
> Click the Layout tab under Table Tools, and then in the Data group, click Repeat Header Rows.

Reviewer: Ok

![track-changes-enable](../../../images/rich-text-editor-track-changes-enable.png)

> When Track Changes is turned on, changes are indicated by red lines in the margin.

Reviewer: ```suggestion
When Track Changes is turned on, red lines in the margin indicate the changes.
```

![IMAGE](../../../images/rich-text-editor-track-changes-simple-markup.png)

> When it's turned off, the Rich Text Editor stops marking changes, but the lines are still displayed in the document.

Reviewer: ```suggestion
When you turn of Track Changes, the Rich Text Editor does not mark changes but displays the lines in the document.
```

Reviewer: Which 'lines' do you refer to?

Author: The lines from the previous sentence. 
> When Track Changes is turned on, changes are indicated by red lines in the margin.
The image shows them too.

Reviewer: Ok

## View Changes

Use the **Display for Review** drop-down list to specify how to display changes in the document. On the **Review** [tab](text-editor-ui/ribbon-interface.md), in the **Tracking** group, open the **Display for Review** list. s

![IMAGE](../../../images/rich-text-editor-track-changes-display-for-review.png)

Select one of the following modes:

* **All Markup** - displays revisions in details.
    ![IMAGE](../../../images/rich-text-editor-track-changes-all-markup.png)

* **Simple Markup** - displays revisions indicated by a red line in the margin.
    ![IMAGE](../../../images/rich-text-editor-track-changes-simple-markup.png)

> * **No Markup** - displays the document without any visible revisions (as if all revisions are accepted).

Reviewer: "any visible revisions" Why 'visible'?

Author: Because changes are in the document, but they are not visible (there are no markup)

Reviewer: Ok

    ![IMAGE](../../../images/rich-text-editor-track-changes-no-markup.png)

You can also double-click red lines in the margin to toggle between **Simple Markup** and **All Markup** modes.

>[!NOTE]
> Rich Text Editor does not display changes in the Reviewing Pane or in balloons.

Use **Next** and **Previous** buttons on the **Changes** ribbon group to switch between revisions.

![IMAGE](../../../images/rich-text-editor-track-changes-previous-next.png)

### View Changes from Specific Authors

On the **Review** [tab](text-editor-ui/ribbon-interface.md), in the **Tracking** group, open the **Show Markup** list. Select users whose changes you wish to display in the **Reviewers** list.

![IMAGE](../../../images/rich-text-editor-track-changes-reviewers.png)

## Lock Tracking

> You can provide a password to prevent users from disabling Track Changes. On the **Review** [tab](text-editor-ui/ribbon-interface.md), in the **Tracking** group, click **Lock Tracking** in the **Track Changes** drop-down list.

Reviewer: ```suggestion
You can set a password that prevents users from disabling Track Changes. In the **Tracking** group on the **Review** [tab](text-editor-ui/ribbon-interface.md), click **Lock Tracking** in the **Track Changes** drop-down list.
```
